src/test/java/com/softserveinc/ita/HomePageTest.java

```diff
@@ -21,6 +26,9 @@ public void verifyLanguageSwitchingTest() {
                 .isTrue();
     }
 
+    protected ProductPage productPage = new ProductPage();
```

Comment:
init this pages in every test where they are needed

between tests should not be any variable or method